forc-pkg/src/manifest.rs

```diff
@@ -608,6 +609,7 @@ impl BuildProfile {
             include_tests: false,
             json_abi_with_callpaths: false,
             error_on_warnings: false,
+            reverse_errors: true,
```

Comment:
```suggestion
            reverse_errors: false,
```
I think we should not reverse the errors by default even if we are using a debug profile. This is pretty specialized feature and users might be surprised if errors are reverted out of the box. Users can pass the flag if they want to have it explicitly.

Also this is why some/all of the should fail tests failing as we are looking for specific errors with them reversing the order breaks the tests.

Reply:
Does `forc build --reverse-errors` work, or does this only work when creating a custom build profile?

Reply:
Willing to approve once this is resolved. I think it would be okay to keep this as a feature that you have to specify in a build profile, since it's likely that those who would use it would always want to use it. I think it would be redundant to have both, as it would defeat the purpose of having the option for custom build profiles. If there is a flag it may lead to people using it under a command alias, instead.

Reply:
cc @nfurfaro thoughts?

Reply:
I've never used a build profile yet... a flag would be sufficient for me, but don't see any harm in supporting both.